_alerts/2017-07-12-potential-split.md

```diff
@@ -12,20 +12,39 @@ bannerclass: "warning"
 ---
 {% assign start='<span class="date">2017/08/01 00:00 UTC</span>' %}
 
-*Last updated: <span class="date">2017/07/12 12:00 UTC</span>.  This
+*Last updated: <span class="date">2017/07/13 11:21 UTC</span>.  This
 page will be updated when new information becomes available.  See the
 [list of updates][].*
 
-At {{start}}, Bitcoin [confirmation scores][] may become unreliable for
-an unknown length of time.  This means that any bitcoins you receive
-after that time may later disappear from your wallet or be a
-type of bitcoin that other people will not accept as payment.
-
-Once the situation is resolved, confirmation scores will either
-automatically return to their normal reliability or there will be two
-(or more) competing versions of Bitcoin.  In the former case, you may
-return to using Bitcoin normally; in the later case, you will need to
-take extra steps in order to begin safely receiving bitcoins again.
+## Chain split
+
+Since Bitcoin's inception, the predominant software that has been used across
+the network has been a program called Bitcoin Core. During this time, Bitcoin has
+grown immensely. The network has facilitated billions of transactions for
```

Comment:
My bad, I was looking at the wrong chart, total value of transactions - thanks for catching. Fixed.